File: Competitive Programming/Data Structure & Algorithm/Recursion/fibonacci.py
def fib(n):
    if n==0:
        return 0
    elif n==1:
        return 1
    return fib(n-1) + fib(n-2)

n = int(input("Enter nth fibonacci number: "))
print(f'{n}th fibonacci number is: {fib(n)}')

File: Competitive Programming/Data Structure & Algorithm/Recursion/fibonacci.cpp
#include <bits/stdc++.h>
using namespace std;
int fibo(int n)
{
    // Base cases
    if (n == 0)
    {
        return 0;
    }
    else if (n == 1)
    {
        return 1;
    }

    // Recursive case
    return fibo(n - 1) + fibo(n - 2);
}

int main()
{
    // Get the number from the user.
    int n;
    cout << "Enter a number: ";
    cin >> n;

    // Print the Fibonacci number.
    cout << "The Fibonacci number for " << n << " is " << fibo(n) << endl;
    return 0;
}